.github/actions/gradle-setup/action.yml

```diff
@@ -3,41 +3,41 @@ description: "Setup Gradle"
 runs:
```

Comment:
Temporary change for testing with CICD. 
This file will be restored when https://github.com/eclipse-dataspaceconnector/DataSpaceConnector/pull/1792 will be merged

Reply:
Are you planning to (perhaps cherry pick and)publish an EDC snapshot?